geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/delta/SetTimestampDeltaUnitTest.java

```diff
@@ -23,18 +23,19 @@
 import java.io.DataOutputStream;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
-import org.apache.geode.redis.internal.data.delta.ReplaceByteArrayAtOffset;
-
-public class DeltaClassesJUnitTest {
+import org.apache.geode.redis.internal.data.RedisString;
+import org.apache.geode.test.junit.runners.GeodeParamsRunner;
 
+@RunWith(GeodeParamsRunner.class)
```

Comment:
This test is not parameterised, so this annotation can be removed.

Reply:
Cleared.